dazzle-rewrite/chunks/lang-go/chunk.yaml

```diff
@@ -1,4 +1,19 @@
 variants:
+  - name: "1.17"
```

Comment:
We should write a test that inspects the version of Go.

Reply:
Every chunk has only one test file. So it is tricky to verify the version. What we can do is to test if the version is one of `x,y,z,a,b,c...` variants.

Reply:
Added in a new commit